app/views/contact_us/contacts/new.html.erb

```diff
@@ -16,36 +16,31 @@
       <fieldset class="side-by-side">
         <% if ContactUs.require_name %>
           <div class="form-input">
-            <label for="contact_us_contact_name"><%= _('Name') %>*</label>
-            <input type="text" id="contact_us_contact_name" name="contact_us[contact_name]" 
-                   class="left-indent required input-large" value="<%= current_user.nil? ? '' : current_user.name(false) %>"<%= current_user.nil? ? '' : 'disabled="true"' %> />
+            <%= f.label(:name, _('Name')+'*') %>
```

Comment:
You can get the '*' character by adding `class: 'required'` for these labels. Its fine to wait though for the final push with the recaptcha fixes.